packages/lisk-p2p/src/peer.ts

```diff
@@ -26,32 +25,29 @@ export interface IPeerConfig {
 }
 
 export class Peer {
-	private height: number;
-	private ip: string;
-	private wsPort: number;
+	private _height: number;
+	private readonly _ip: string;
+	private _wsPort: number;
```

Comment:
why wsPort is not readonly?

Reply:
I think I need to discuss with Jon about which fields should be kept readonly, as last time he only mentioned `ip`